### PR TITLE
[CAS-792] Feature - Enable disable replies

### PR DESCRIPTION
**Pull request by leandroBorgesFerreira**
[CAS-792](https://stream-io.atlassian.net/browse/CAS-792)

### Checklist

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [x] PR targets the `develop` branch
- [x] Changelog updated with client-facing changes
- ~[ ] New code is covered by unit tests~
- ~[ ] Comparison screenshots added for visual changes~
- [x] Reviewers added

### CODE REVIEW

File: UNRELEASED_CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 ### 🐞 Fixed
 Group channels with 1<>1 behaviour the same way as group channels with many users
 It is not possible to remove users from distinct channels anymore.
+Configure enable/disable of replies using XML in `MessageListView`
```
Comment (adasiewiczr) [inline UNRELEASED_CHANGELOG.md:5]: Shouldn't this be moved to `Added` section?

**Review by adasiewiczr — APPROVED** [review]

File: stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/options/message/internal/MessageOptionsView.kt [hunk 2]
```diff
@@ -59,7 +63,11 @@ internal class MessageOptionsView : FrameLayout {
     private fun configureMineMessage(configuration: Configuration, syncStatus: SyncStatus) {
         val iconsTint = configuration.iconsTint
 
-        binding.replyTV.configureListItem(configuration.replyIcon, iconsTint)
+        if (configuration.replyEnabled) {
```
Comment (adasiewiczr) [inline stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/options/message/internal/MessageOptionsView.kt:66]: Would be nice to extract this into a method and reuse inside `configureTheirsMessage`

**Comment by adasiewiczr** [comment]
What about enabling/disabling replies using member methods?

File: UNRELEASED_CHANGELOG.md (hunk 1 above)
Comment (bychkovdmitry) [reply UNRELEASED_CHANGELOG.md:5]: Agree, added section is better. Also, would be nice to mention what attribute was added.

File: stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/MessageListView.kt [hunk 3]
```diff
@@ -515,6 +517,7 @@ public class MessageListView : ConstraintLayout {
         messageOptionsConfiguration = MessageOptionsView.Configuration(
             iconsTint = iconsTint,
             replyIcon = replyIcon,
+            replyEnabled = replyEnabled,
```
Comment (bychkovdmitry) [inline stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/MessageListView.kt:520]: Maybe all "enabled" attributes at the bottom? Or we can move the "copyTextEnabled" field above "copyIcon" later.

**Review by bychkovdmitry — APPROVED** [review]
Looks good, but as @adasiewiczr said, member method is also needed.

**Pull request merged**